src/utils/api.js

```diff
@@ -6,15 +6,44 @@ const instance = axios.create({
 
 function getAllArticles (){
   return instance.get("/articles")
+  .then(({data})=>{
+    const articlesArr = data.articles.sort((a, b)=>{
```

Comment:
does the api not sort the articles for you? 
I' not sure you need them to be arranged by ID, but if you do this should be extracted into its own (fully tested) util function.